citadel-internal-service/tests/common/mod.rs

```diff
@@ -33,12 +33,13 @@ pub fn setup_log() {
     }));
 }
 
-pub struct RegisterAndConnectItems<T: Into<String>, R: Into<String>, S: Into<SecBuffer>> {
+pub struct RegisterAndConnectItems<T: Into<String>, S: Into<SecBuffer>> {
     pub internal_service_addr: SocketAddr,
     pub server_addr: SocketAddr,
     pub full_name: T,
-    pub username: R,
+    pub username: T,
```

Comment:
Usually, re-using type parameters between fields is not the best practice for flexibility. Whereas before, I could use a `&'static str` for the full name and a `String` for the username, now, because both `full_name` and `username` are `T`, they have to be the same type.

Reply:
So, I'd revert this back to what it was.